.gitignore

```diff
@@ -18,7 +18,6 @@
 .env.development.local
 .env.test.local
 .env.production.local
-src/holofuel/__integration_tests__/holo/snapshots/*
```

Comment:
@JettTech - this line won't actually create the snapshots directory. See the `.gitignore` file I added for an example of a way to do that.